workspaces/acs/.changeset/public-sides-knock.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage-community/plugin-acs': patch
```

Comment:
Does it make sense to drop this changeset?

Reply:
Yeah I can definitely remove it.